python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -387,6 +391,39 @@ def get_asset_keys_to_resource_props(
     }
 
 
+def get_updated_cli_invocation_params_for_context(
+    context: Union[OpExecutionContext, AssetExecutionContext],
+    manifest: Mapping[str, Any],
+    dagster_dbt_translator: "DagsterDbtTranslator",
+):
+    assets_def: Optional[AssetsDefinition] = None
+    with suppress(DagsterInvalidPropertyError):
```

Comment:
can we branch this behavior based off of a type check of OpExecutionContext vs AssetExecutionContext? The suppressing bit is a bit odd

Reply:
I've completely removed the suppressing in [4a9ac16](https://github.com/dagster-io/dagster/pull/29650/commits/4a9ac16a113b92be5d856f24c4867e8d97c70f5d)